config/deploy/production.rb.sample

```diff
@@ -7,8 +7,22 @@ server 'hostname_or_ip', user: 'zartan', roles: %w{app db web}
 # Deploy from a branch other than master:
 # set :branch, 'another-branch'
 
-# Add an environment variable:
-# set :default_env, -> { fetch(:default_env).merge( var_name: 'value' ) }
+set :repo_url, 'git@YOUR_GIT_SSH_URL.git'
```

Comment:
If we're setting the `:repo_url` in here, the corresponding line should probably be removed from config/deploy.rb